edx_django_utils/management/tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for Student Management Commands."""
```

Comment:
```suggestion
"""Tests for Management Commands."""
```
These new management commands (as well as future commands that could live here) are not student-specific.